README.md

```diff
@@ -41,11 +41,29 @@ Description for VM
 **Supported properties:**
 * `libvirt_auth`: connection url, by default: `qemu:///system`
 * `backup_dir`: directory for save backups, by default: `./`
+* `params`: params used for create object, useful for embeded template.
+  * `use_external_resource`: (optional) Use external object. The default is
+    `false`.
+  * `resource_id`: (optional) Used to identify the object when
+    `use_external_resource` is true.
+  * `vcpu`: CPU count
```

Comment:
why are `resource_id` and `use_external_resource` params instead of properties? (In 5.0 these will be required properties on all resources.

Reply:
Moved to top of properties.